### PR TITLE
Added a random task and moved all other tasks to /tasks/all

### PR DESCRIPTION
**Pull request by kostiak**
Creating a new draft PR for random task post-refactor of the tasks.

Notes:

- The design needs some more work
- All tasks moved to the `/tasks/all` route, we need to decide what to do with it. Not linked anywhere for now for internal/testing use.
- Submitting labeling tasks is not working right 
- - Found temporary workaround
- Need to reset slider values in Label

### CODE REVIEW

**Comment by kostiak** [comment]
@fozziethebeat take a look at this when you can.

I'm thinking maybe merge it and open sperate issues

File: website/src/pages/tasks/random.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    return <Container className="p-6 text-center text-gray-800">No tasks found...</Container>;
+  }
+
+  return (
+    <>
+      <Head>
+        <title>Random Task</title>
+        <meta name="description" content="Random Task." />
+      </Head>
+      <Task tasks={tasks} trigger={trigger} mutate={reset} />
```
Comment (fozziethebeat) [inline website/src/pages/tasks/random.tsx:31]: Someone changed this part a little bit.  It now takes `{tasks[0].task}`

Also, to fix some changes that just got merged in, there's a new property: `frontendId={tasks[0].id]}`

**Review by fozziethebeat — COMMENTED** [review]
I agree. We should add random first, then remove and hide everything else.

From some PRs that got merged in, there's some small changes I think you need to handle.

File: website/src/pages/tasks/random.tsx (hunk 1 above)
Comment (fozziethebeat) [reply website/src/pages/tasks/random.tsx:31]: For reference: https://github.com/LAION-AI/Open-Assistant/pull/593 fixes a few issues that with what's at HEAD.

Comment (kostiak) [reply website/src/pages/tasks/random.tsx:31]: Got it, thanks. I'll take a look at it when I can (bit of a busy day at work today).

Comment (fozziethebeat) [reply website/src/pages/tasks/random.tsx:31]: Sounds good! Most of the fixes will be needed after I merge my PR anyway.

Comment (kostiak) [reply website/src/pages/tasks/random.tsx:31]: Failed to rebase... I'll close this PR and open a new one I guess...

**Comment by fozziethebeat** [comment]
I ran into a similar issue with one of my PRs.  I managed to rollback the
rebase changes and the fix everything with a simple merge.

**Pull request closed without merging**